spec/features/data_access/service_options_shapefile_spec.rb

```diff
@@ -0,0 +1,37 @@
+# EDSC-2084
+# Disable the use-shapefile checkbox if there was no upload during search
+
+require 'rails_helper'
+require 'rake'
+
+describe 'Service options order with use shapefile field', pending_updates: true, single_granule: true do
```

Comment:
Will need to remove `, pending_updates: true, single_granule: true` in order for this test to run